components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -627,24 +627,36 @@ subroutine ocn_relativeVorticity_circulation(relativeVorticity, circulation, nor
       !$omp end do
 #endif
 
+      ! no-slip = 0.0
+      ! free-slip = 1.0
+      ! partial-slip in-between
+      wall_slip_factor = config_wall_slip_factor
+
 #ifdef MPAS_OPENACC
       !$acc parallel loop &
-      !$acc          present(circulation, relativeVorticity, areaTriangle, edgesOnVertex, &
-      !$acc                  maxLevelVertexBot, dcEdge, normalVelocity, edgeSignOnVertex, &
-      !$acc                  minLevelVertexTop) &
-      !$acc          private(invAreaTri1, iVertex, iEdge, k, r_tmp)
+      !$acc          present(circulation, relativeVorticity, edgesOnVertex, &
+      !$acc                  maxLevelVertexBot, dcEdge, normalVelocity, &
+      !$acc                  minLevelVertexTop, areaTriangle, edgeSignOnVertex) &
+      !$acc          private(invAreaTri1, i, iEdge, k, r_tmp, wall_slip_factor)
```

Comment:
```suggestion
      !$acc          private(invAreaTri1, i, iEdge, k, r_tmp)
```
This change was made prior to testing. At least on some compilers, including `wall_slip_factor` in `private` results in it being assigned to 0 in the loop.

Reply:
Fixes a bug of mine.